test/NuGet.Core.Tests/NuGet.Packaging.Test/ContentModelTests/PatternExpressionMatchTests.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.ContentModel;
+using NuGet.ContentModel.Infrastructure;
+using Xunit;
+
+namespace NuGet.Client.Test
+{
+    public class PatternExpressionMatchTests
+    {
+
+        
+        [Fact]
+        public void LiteralSegment_OnlyShouldMatch()
+        {
+            // match literal segments given a pattern
+            Dictionary<string, ContentPropertyDefinition> propertyDefinitions =
+                new Dictionary<string, ContentPropertyDefinition>() {
+                {"type", new ContentPropertyDefinition("text") }
+            };
+            var pattern = new PatternDefinition("content/file.txt");
+            var expression = new PatternExpression(pattern);
+            var path = "content/file.txt";
```

Comment:
Avoid string duplication.

```suggestion
            var path = "content/file.txt";
            var pattern = new PatternDefinition(path);
            var expression = new PatternExpression(pattern);
```